server/src/main/java/org/elasticsearch/search/aggregations/bucket/BucketsAggregator.java

```diff
@@ -163,4 +165,24 @@ public final void close() {
         }
     }
 
+    @Override
+    public Aggregator resolveSortPath(AggregationPath.PathElement next, Iterator<AggregationPath.PathElement> path) {
+        if (this instanceof SingleBucketAggregator) {
```

Comment:
I'm not super thrilled about this, but it is how it used to work and I wanted to keep this change fairly contained.

Reply:
:+1: